src/journal/journald-syslog.c

```diff
@@ -145,6 +150,11 @@ void server_forward_syslog(Server *s, int priority, const char *identifier, cons
         if (LOG_PRI(priority) > s->max_level_syslog)
                 return;
 
+        if (s->facilities_store) {
+                if (!set_contains(s->facilities_store, UINT_TO_PTR(LOG_FAC(priority))))
+                        return;
+        }
+
```

Comment:
Why this is needed? It seems that `FacilitiesStore` should not affect what messages are forwarded to `syslog`.